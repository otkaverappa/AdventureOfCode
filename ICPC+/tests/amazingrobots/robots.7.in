8 8
########
##.#...#
##.###..
#....X.#
####...#
#......#
#......#
########
4
3 3 2 S
6 5 3 W
5 5 3 E
2 7 4 S
8 8
########
###....#
#...####
..###..#
#...#.##
##.X##.#
#.......
########
3
3 2 3 E
5 2 2 N
7 7 2 N
