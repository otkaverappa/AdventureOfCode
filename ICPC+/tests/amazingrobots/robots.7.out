13
E
W
E
N
E
E
S
S
E
E
E
E
E
